### DISCUSSION TITLE
Future ideas for Clockwork!

### DISCUSSION THREAD

**Discussion by adonzone** [red-stone-network/clockwork#54] [Ideas]
Some future ideas I have for clockwork are:
- Simplistic bubble logos (as attached below)
- Booting intro
- More Backgrounds
- Better Clockwork Launcher (I'm working on one)

**Comment by adonzone** [comment]
forgot to attach the bubble logos
![438fef28311610cd49c66c06ac1a94e9](https://github.com/red-stone-network/clockwork/assets/56778024/06bc2b5d-3958-4e7d-8a38-b6a808e9458b)
Something like this

**Comment by adonzone** [comment]
ai redesign (thanks bing) 
![_7a706af7-b356-408d-9ace-93a8d33bf47d](https://github.com/red-stone-network/clockwork/assets/56778024/814bb6e4-4d8b-43af-b1be-66ededf427bf)
![_86b77736-9604-46af-80bf-3ca5e2ced94d](https://github.com/red-stone-network/clockwork/assets/56778024/1db2398e-fbd0-43f6-9192-75ebf87a0bba)

**Reply by Ender-Ghost-Craft** [reply]
maybe add my (crazy cool) ideas? I don't care if you give credit

**Reply by adonzone** [reply]
yeah we should... im not sure how hard it woult be...

**Comment by wow2302** [comment]
I think addding a internet changer would be cool. So you wont have to worry about the school wifi blocking clockwork apps from working. It would be included in setting or sum so school wifi wont be a problem for people who cant connect to other wifi

**Reply by ToasterPanic** [reply]
Please explain, what is an "internet changer"?

**Reply by Ender-Ghost-Craft** [reply]
something to bypass internet blocks

**Reply by adonzone** [reply]
yeah i have internet blocks... but you can bypass them using proxies if I'm correct? you can find one at 1.1.1.1/dns/

**Reply by wow2302** [reply]
When I say a internet changer its like something that will go to random d.n.ses (any) that can throw off, or mess up the school wifis filter block which causes some clockwork apps to act up. D.N.Ses dont work for me because my school blocked changing the wifi, but if it were done in a app they cant do anything about it (of course unless they find clockwork but im not dumb) so yeah.

**Reply by wow2302** [reply]
This would mainly be used to bypass the wifi blocks. Since the school restricted changing wifi options for those who cant.
